File: Luv Babbar dsa/Extra/Infytq/Self Sufficient.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin>>n;
    vector<int>earn(n,0);
    for(int i=0;i<n;i++)
    {
        cin>>earn[i];
    }
    
    vector<int>cost(n,0);
    for(int i=0;i<n;i++)
    {
        cin>>cost[i];
    }
    
    int mini=0;
    for(int i=0;i<n;i++)
    {
        if(cost[i]>earn[i])
        {
            mini+=(cost[i]-earn[i]);
        }
        else if(cost[i]<earn[i])
        {
            mini-=(earn[i]-cost[i]);
        }
    }
    cout<<mini<<endl;
}

File: Luv Babbar dsa/Extra/Infytq/SelfSufficient.java
// tc:O(n)
// sc:o(1)
import java.util.*;
public class Main
{
    public static void main(String args[])
    {
        Scanner sc =new Scanner(System.in);
        int n=sc.nextInt();
        
        int[] earn=new int[n];
        for(int i=0;i<n;i++)
        {
            earn[i]=sc.nextInt();
        }
        
        int[] cost=new int[n];
        for(int i=0;i<n;i++)
        {
            cost[i]=sc.nextInt();
        }
        
        int mini=0;
        for(int i=0;i<n;i++)
        {
            mini+=cost[i]-earn[i];
        }
        if(mini<0)
        {
            mini=0;
        }
        System.out.println(mini);
    }
}
